osu.Game/Graphics/UserInterface/OsuTextBox.cs

```diff
@@ -0,0 +1,48 @@
+﻿//Copyright (c) 2007-2016 ppy Pty Ltd <contact@ppy.sh>.
+//Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.UserInterface;
+using osu.Framework.Input;
+using OpenTK.Graphics;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public class OsuTextBox : TextBox
+    {
+        public OsuTextBox()
+        {
+            Height = 40;
+            TextContainer.Height = 0.6f;
+            CornerRadius = 5;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colour)
+        {
+            BorderColour = colour.Yellow;
+        }
+
+        protected override bool OnFocus(InputState state)
+        {
+            BorderThickness = 3;

```

Comment:
we will need to make a transform for this in the future (it should fade eventually).